components/menu/menu-item-link.js

```diff
@@ -33,6 +38,36 @@ class MenuItemLink extends MenuItemMixin(LitElement) {
 	}
 
 	static get styles() {
+		if (newWindowIconEnabled) return [ super.styles, menuItemStyles,
+			css`
+				:host {
+					display: block;
+					padding: 0;
+				}
+
+				:host > a {
+					align-items: center;
+					color: inherit;
+					display: flex;
+					line-height: 1rem;
+					outline: none;
+					overflow-x: hidden;
+					padding: 0.75rem 1rem;
+					text-decoration: none;
+				}
+
+				:host([target="_blank"]) .d2l-menu-item-text {
+					align-self: baseline;
+					flex: 0 1 auto;
+				}
+
+				#new-window {
+					align-self: baseline;
+					flex: auto;
+				}
+			`
+		];
+
 		return [ menuItemStyles,
```

Comment:
Hopefully whoever is cleaning up would realize this block should be removed during cleanup, but often times we add a comment like:
```javascript
// remove this block when cleaning up GAUD-8295-menu-item-link-new-window-icon
```